crates/gpui/src/platform.rs

```diff
@@ -6,6 +6,9 @@
 mod app_menu;
 mod keystroke;
 
+#[cfg(not(target_os = "macos"))]
+mod common;
```

Comment:
I don't know if this is the best choice. I was considering adding a module called `cosmic_text` instead of 'common' and haing a `CosmicTextSystem` instead of `CommonTextSystem`. Thoughts?

```suggestion
#[cfg(not(target_os = "macos"))]
mod cosmic_text;
```